src/main/java/io/github/kawamuray/wasmtime/ExportType.java

```diff
@@ -0,0 +1,55 @@
+package io.github.kawamuray.wasmtime;
+
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.experimental.Accessors;
+
+@Accessors(fluent = true)
+@AllArgsConstructor(access = AccessLevel.PACKAGE)
+public class ExportType {
```

Comment:
Now it turns out at original wamstime, there's just `ExternType` which is used for both imports and exports. As the definition of this class is exactly the same as `ImportType`, let's unify them and rename it to `ExternType`?

Reply:
I thought about that initially too, however i'm conflicted because of compatibility issues, because 0.10 was already released. Should i ignore it or do you have a better Idea?

Reply:
Not sure if this is the standard in many projects, but at least for few projects I've seen, they never guarantee compatibility among minor versions while the major version stays 0, and I'm basically the fan of that way so I think it's just fine :)

Reply:
I mean Semantic Versioning defines it like that as well. However there is no mention anywhere, that this Project uses it.

So better safe than sorry.